staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -39,8 +39,10 @@ import (
 )
 
 func NewEventFromRequest(req *http.Request, level auditinternal.Level, attribs authorizer.Attributes) (*auditinternal.Event, error) {
+	now := metav1.NewTime(time.Now())
 	ev := &auditinternal.Event{
-		Timestamp:  metav1.NewTime(time.Now()),
+		ObjectMeta: metav1.ObjectMeta{CreationTimestamp: now},
```

Comment:
we have to update this for every stage.

Reply:
only `CreationTimestamp`, not `Timestamp`.

Reply:
> we have to update this for every stage.
 
This is not addressed, I think.